### PR TITLE
content: draft: source-track: rename SCP to SCS, replace issue links with project board links. 

### PR DESCRIPTION
**Pull request by zachariahcox**
fixes: https://github.com/slsa-framework/slsa/issues/1128

(cleaned up version of https://github.com/slsa-framework/slsa/pull/1166)

This change is in response to the 9.30 slsa specification meeting on this topic.

A SCS is the full suite of services and ideas relied upon by the organization to create source revisions.

VCS stuff should mostly fall out of the discussion

Repositories can be used as the concept used when we need to talk about authN and authZ w.r.t. authentic contributions.

### CODE REVIEW

File: docs/spec/draft/source-requirements.md [hunk 1]
```diff
@@ -26,17 +20,17 @@ Consumers can examine the various source provenance attestations to determine if
...
-| Organization | A collection of people who collectively create the Source. Examples of organizations include open-source projects, a company, or a team within a company. The organization defines the goals and methods of the repository.
-| Repository / Repo | A uniquely identifiable instance of a VCS hosted on an SCP. The repository controls access to the Source in the version control system. The objective of a repository is to reflect the intent of the organization that controls it.
-| Branch | A named pointer to a revision. Branches may be modified by authorized actors. In git, cloning a repo will download all revisions in the history of the "default" branch to the local machine. Branches may have different security requirements.
-| Change | A set of modifications to the source in a specific context. Can be proposed and reviewed before being accepted.
+| Source Control System (SCS) | A service or suite of services (self-hosted or SaaS) relied upon by the organization to produce new revisions of the source. GitHub and GitLab are examples, as are combinations of tools like Gerrit code reviews with GitHub Repositories.
```
Comment (adityasaky) [inline docs/spec/draft/source-requirements.md:26]:
```suggestion
| Source Control System (SCS) | A suite of tools and services (self-hosted or SaaS) relied upon by the organization to produce new revisions of the source. The role of the SCS may be fulfilled by a single service (e.g., GitHub / GitLab) or rely on a combination of services (e.g., GitLab with Gerrit code reviews, GitHub with OpenSSF Scorecard, etc.).
```

I took a stab at broadening the definition a bit to indicate that the suite of services can include things like Scorecard.

**Review by adityasaky — COMMENTED** [review]
I think this looks great! I have one suggestion to broaden the definition of the SCS.

**Review by TomHennen — APPROVED** [review]
Looks good, thanks!

File: docs/spec/draft/source-requirements.md (hunk 1 above)
Comment (zachariahcox) [reply docs/spec/draft/source-requirements.md]: thanks!

**Review by mlieberman85 — APPROVED** [review]
👍

**Comment by TomHennen** [comment]
Two maintainers seems good enough for a draft.

Note: I've updated the title to indicate this is a draft.

**Pull request merged**